crypto/util.go

```diff
@@ -0,0 +1,8 @@
+package neofscrypto
+
+import "encoding/hex"
+
+// StringifyKeyBinary returns string with HEX representation of source
```

Comment:
I suggest to also mention that format can be changed and it's unsafe to rely on it beyond human-readable output.

Reply:
Updated.